apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -3778,6 +3796,80 @@ sc_ws_close_mutual_(Config, Closer) when Closer =:= initiator
     {ok, 200, #{<<"transactions">> := []}} = get_pending_transactions(),
     ok.
 
+sc_ws_close_solo_(Config, Closer) when Closer =:= initiator;
+                                       Closer =:= responder ->
+    ct:log("ConfigList = ~p", [Config]),
+    #{initiator := #{pub_key  := IPubKey,
+                     priv_key := IPrivKey},
+      responder := #{pub_key  := RPubKey,
+                     priv_key := RPrivKey}} = proplists:get_value(participants,
+                                                                      Config),
+    {IStartB, RStartB} = channel_participants_balances(IPubKey, RPubKey),
+    #{initiator := IConnPid,
+      responder := RConnPid} = proplists:get_value(channel_clients, Config),
+    ok = ?WS:register_test_for_channel_events(IConnPid, [sign, info, on_chain_tx]),
+    ok = ?WS:register_test_for_channel_events(RConnPid, [sign, info, on_chain_tx]),
+
+    CloseSolo =
+        fun(CloserConn, CloserPrivKey, OtherConn, OtherPrivKey) ->
+                ws_send_tagged(CloserConn, <<"channels.close_solo">>, #{}, Config),
```

Comment:
Where are these methods meant to be documented? In [intended usage of channels API](https://github.com/aeternity/protocol/blob/8b2724d14d8b4878b6b2c74850ec3f7f7716aeb2/node/api/channels_api_usage.md)? It seems doc requires integration.

Reply:
Working on the documentation additions.